src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml

```diff
@@ -244,7 +244,7 @@
           VerticalContentAlignment="Center"
           Checked="_checkboxVulnerabilities_Checked"
           Unchecked="_checkboxVulnerabilities_Unchecked"
-          Visibility="Visible"
+          Visibility="Hidden"
```

Comment:
I would expect `Collapsed` unless there's a penalty on redrawing the controls on resizing.
Did you test a small PM UI window and see a problem with toggling between Collapsed/Visible?

Reply:
Yes, that's the difference between Collapsed/Hidden. However, I don't see any reason why the Search textbox should shrink in your example as your screen has plenty of blank space in the Top Panel. That probably means the containing layout control is too small, so it's making the vulnerability checkbox consume some of its space.

I think we need to fix this to use the proper width, and only resize controls when necessary (based on the PM UI's width).
It's reasonable to merge this then fix that separately since 1. the hidden behavior is already released before this PR and 2. the collapsed behavior will need to be tested well (different resolutions, DPI scaling, etc).